hubble/src/indexer/mod.rs

```diff
@@ -38,20 +38,30 @@ pub struct FinalizerConfig {
     // how many blocks to wait until a block is considered finalized (ie. there should be no reorgs).
     // compensates for height differences between rpcs
     // default: 5
-    #[serde(default)]
+    #[serde(default = "FinalizerConfig::finalizer_config_default_delay_blocks")]
     pub delay_blocks: usize,
     // reload all block data after a block is considered finalized
     // compensates for rpcs returning inconsistent results for non-finalized blocks.
     // default: true
-    #[serde(default)]
+    #[serde(default = "FinalizerConfig::finalizer_config_default_reload")]
     pub reload: bool,
 }
 
+impl FinalizerConfig {
+    pub fn finalizer_config_default_delay_blocks() -> usize {
+        5
+    }
+
+    pub fn finalizer_config_default_reload() -> bool {
+        true
+    }
```

Comment:
minor nit: no need to prefix these functions with `finalizer_config_*`

Reply:
fixed (and squashed)